{
    "output": "",
    "errors": "F..F.FFF\n======================================================================\nFAIL: test_empty_haystack (__main__.TestSolution.test_empty_haystack)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File \"C:\\Users\\Serge\\Desktop\\GPT-Testing\\DataSets\\DataSet-Old\\LeetCodeTasks - Testing\\Tests\\28_find-the-index-of-the-first-occurrence-in-a-string\\test.py\", line 57, in test_empty_haystack\n    self.assertEqual(self.solution.strStr(\"\", \"a\"), -1)\nAssertionError: 0 != -1\n\n======================================================================\nFAIL: test_example2 (__main__.TestSolution.test_example2)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File \"C:\\Users\\Serge\\Desktop\\GPT-Testing\\DataSets\\DataSet-Old\\LeetCodeTasks - Testing\\Tests\\28_find-the-index-of-the-first-occurrence-in-a-string\\test.py\", line 23, in test_example2\n    self.assertEqual(self.solution.strStr(\"leetcode\", \"leeto\"), -1)\nAssertionError: 0 != -1\n\n======================================================================\nFAIL: test_needle_larger_than_haystack (__main__.TestSolution.test_needle_larger_than_haystack)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File \"C:\\Users\\Serge\\Desktop\\GPT-Testing\\DataSets\\DataSet-Old\\LeetCodeTasks - Testing\\Tests\\28_find-the-index-of-the-first-occurrence-in-a-string\\test.py\", line 63, in test_needle_larger_than_haystack\n    self.assertEqual(self.solution.strStr(\"abc\", \"abcd\"), -1)\nAssertionError: 0 != -1\n\n======================================================================\nFAIL: test_single_char_haystack (__main__.TestSolution.test_single_char_haystack)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File \"C:\\Users\\Serge\\Desktop\\GPT-Testing\\DataSets\\DataSet-Old\\LeetCodeTasks - Testing\\Tests\\28_find-the-index-of-the-first-occurrence-in-a-string\\test.py\", line 31, in test_single_char_haystack\n    self.assertEqual(self.solution.strStr(\"a\", \"b\"), -1)\nAssertionError: 0 != -1\n\n======================================================================\nFAIL: test_single_char_needle (__main__.TestSolution.test_single_char_needle)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File \"C:\\Users\\Serge\\Desktop\\GPT-Testing\\DataSets\\DataSet-Old\\LeetCodeTasks - Testing\\Tests\\28_find-the-index-of-the-first-occurrence-in-a-string\\test.py\", line 43, in test_single_char_needle\n    self.assertEqual(self.solution.strStr(\"abc\", \"d\"), -1)\nAssertionError: 0 != -1\n\n----------------------------------------------------------------------\nRan 8 tests in 0.002s\n\nFAILED (failures=5)\n",
    "returncode": 1
}